gi_loadouts/type/file/arti.py

```diff
@@ -129,7 +129,11 @@ def make_artifile_from_yaml(objc: dict) -> ArtiFile:
             stat_d=ATTR(stat_name=__revmap__[objc["stat"]["d"]["name"]], stat_data=objc["stat"]["d"]["data"]),
         )
     except Exception as expt:
-        raise ValueError("Artifact unit data cannot be parsed.") from expt
+        raise ValueError("Artifact stat cannot be identified or unit data cannot be parsed.") from expt
+
+    for indx in ["a", "b", "c", "d"]:
+            if artiobjc.stat_main.stat_name == getattr(artiobjc, f"stat_{indx}").stat_name:
```

Comment:
Please default to four-spaces long indentations. 

```suggestion
    for indx in ["a", "b", "c", "d"]:
        if artiobjc.stat_main.stat_name == getattr(artiobjc, f"stat_{indx}").stat_name:
```

Reply:
Weird :confused: 

Might be caused while using the shortcuts in the IDE.